api/propagators/b3_propagator.go

```diff
@@ -35,7 +33,7 @@ const (
 	B3ParentSpanIDHeader = "X-B3-ParentSpanId"
 )
 
-// B3Propagator that facilitates core.SpanContext
+// B3 that facilitates core.SpanContext
```

Comment:
Please update this documentation to make grammatical sense; e.g. `B3 propagator serializes core.SpanContext to/from B3 Headers.`

Reply:
updated.